src/test/java/api/support/builders/ChangeDueDateRequestBuilder.java

```diff
@@ -0,0 +1,57 @@
+package api.support.builders;
+
+import static org.folio.circulation.resources.ChangeDueDateResource.DUE_DATE;
+
+import java.util.UUID;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import io.vertx.core.json.JsonObject;
+
+public class ChangeDueDateRequestBuilder implements Builder {
+  private final DateTime dueDate;
+  private final String loanId;
+
+  public ChangeDueDateRequestBuilder() {
+    this(null, DateTime.now(DateTimeZone.UTC));
+  }
+
+  private ChangeDueDateRequestBuilder(
+    String loanId, DateTime dueDate) {
```

Comment:
Why are these formatted on a separate line rather than using the standard line splitting?
```suggestion
  private ChangeDueDateRequestBuilder(String loanId, DateTime dueDate) {
```